cirq/google/calibration/phased_fsim_test.py

```diff
@@ -0,0 +1,48 @@
+from cirq.google.calibration.phased_fsim import PhasedFSimCharacterization
```

Comment:
This too is missing copyright.

Reply:
I guess, we should have a script to check the boilerplate.

Reply:
Done